README.md

```diff
@@ -1,112 +1,204 @@
+
+Release lines in "Current" _may_ receive updates to their V8 dependency. This will occur ***only if*** it can be done in a way that is not semver-major and does not break the stable ABI of the current major version. This may be achieved by either receiving an ABI-stable update from upstream or by applying floating patches to the dependency to ensure ABI-stability. Implementation details are left up to the Node.js V8 team.
+
+Upgrades to V8 in a "Current" release line should not occur within two months of that release line moving in to "Active LTS". i.e. upgrades to V8 should not occur from August through to the transition of "Current" into LTS in October.
+
+### Changes acceptable for "Active LTS"
+
+Release lines currently in "Active LTS" will only receive changes that satisfy one or more of the following categories:
+
+1. Bug fixes
+2. Security updates
+3. Non-semver-major npm updates
+4. Relevant documentation updates
+5. Certain performance improvements where the risk of breaking existing applications is minimal
+6. Changes that introduce large amount of code churn where the risk of breaking existing applications is deemed to be very low and where the change in question may significantly ease the ability to backport future changes due to the reduction in "diff noise"
+7. Updates to other dependencies that introduce bug fixes, where members of the LTS Working Group deem such updates very low-risk to stability and security
+
+### Changes acceptable for "Maintenance"
+
+Release lines currently in "Active LTS" will only receive changes that satisfy one or more of the following categories:
 
-New semver-major releases of Node.js are cut from `master` every six months.
-New even-numbered versions (e.g. v6, v8, v10, etc) are cut in April. New 
-odd-numbered versions (e.g. v5, v7, v9) are cut in October.
+1. ***Critical*** bug fixes
+2. ***Critical*** security fixes
+3. Relevant documentation updates
+4. Updates to dependencies that introduce critical bug fixes, where members of the LTS Working Group deem such updates very low-risk to stability and security
 
-When a new *odd-numbered* major release is cut, the previous *even-numbered* 
-major version transitions to the Long Term Support plan.
+### Semver-major changes in LTS
 
-Every major version covered by the LTS plan will be actively maintained for a 
-period of 18 months from the date it enters LTS coverage. Following those 18 
-months of active support, the major version will transition into "maintenance" 
-mode for 12 additional months.
+It is possible that ***critical*** security and bug fixes may be labelled "semver-major".
+Such situations are expected to be rare.
 
-Given this schedule, there will be no more than two active LTS releases at any 
-given time, overlapping for a maximum period of six months.
+Where it is necessary to introduce such "semver-major" changes into a release line covered by LTS.
+The changes will result in an increment of the semver-minor version number only.
```

Comment:
May want to add a short explanation about *why* semver-major bumps in LTS streams are handled as semver-minors... specifically, the Semantic Versioning spec does not provide a mechanism for us to adequately communicate that a semver-major change has occurred within an LTS release line.